documents/0.5.0/terran_unit_mercs_pairing_triggers.yaml

```diff
@@ -0,0 +1,271 @@
+# Original author: Ferin (Pankuu22)
+# Unit & Merc Pairs
+# Mercs will be paired with base unit counterparts to receive upgrade item benefits
+# No_mercs results will exclude Merc item but leaves the base unit's presence in the multiworld to the randomizer

```

Comment:
This line is inaccurate; this trigger either excludes _both_ the unit and the merc, or locks both.

Reply:
Aahh ok that's my bad. Forgot how this file was outdated from the beta SC2 yaml I use that's using the updated triggers, and I didn't update this separate file.